### DISCUSSION TITLE
Multiple options for things being enabled by default

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#172] [Discussion]
My idea is that you can select which extensions should be enabled, so you can have different categories, such as

- None
- Few
- Recommended
- All

So I asked a while ago what exactly can be enabled by default and it's kinda the second category so I think it would be neat to have "Recommended" stuff like the discuss button and the image uploader. What do you think?

**Comment by retronbv** [comment]
Yes, this would be amazing

**Comment by retronbv** [comment]
But how would we decide what is in what

**Comment by NoobTracker** [comment]
Common sense. As @WorldLanguages said about being enabled by default. You recommend stuff like the image uploader, but not a dark theme. Small but useful things are recommended.

**Comment by Explosion-Scratch** [comment]
Sure! This sounds great!

**Comment by Explosion-Scratch** [comment]
Also, maybe merge all addons even if they're experimental but put them in the 'beta' section?

**Comment by NoobTracker** [comment]
@Explosion-Scratch Every single addon? Some work quite well, that's not beta.

**Comment by retronbv** [comment]
We can test which are good and not put them in beta

**Comment by NoobTracker** [comment]
I guess we can

**Comment by WorldLanguages** [comment]
Closed in favor of #318
